src/java.base/unix/native/libnio/ch/UnixFileDispatcherImpl.c

```diff
@@ -45,6 +45,11 @@
 #define fstatvfs64 fstatvfs
 #endif
 
+#if defined(__linux__)
+#include <linux/fs.h>
+#include <sys/ioctl.h>
+#endif
+
```

Comment:
This looks okay, I just wonder if the `#ifdef BLKGETSIZE64` could be replaced with `#ifdef __linux__` to avoid accidents when the set of include files changes.